lib/index.js

```diff
@@ -53,7 +53,11 @@ class PWMetrics {
     const allDone = results.reduce((chain, run, index) => {
       return chain.then(_ => {
         return this.run().then(data => {
+          console.log("Run",index+1 , "of", results.length, "finished successfully");
           results[index] = data;
+        }).catch(error => {
+          console.log("unable to complete run", index+1, "of", results.length, "due to", error.message);
```

Comment:
Done both, please see new commit and program output in the screenshot I commented :)